prijateli_tree/app/routers/games.py

```diff
@@ -185,9 +188,14 @@ def get_previous_answers(
         this_answer = [
             a for a in this_neighbor.answers if a.round == last_round
         ][0]
-        complete_name = (
-            f"{this_neighbor.user.first_name} {this_neighbor.user.last_name}: "
-        )
+
+        # Check if names are hidden
+        if names_hidden:
+            player_id = this_neighbor.user.id
+            complete_name = f"Player {player_id}: "
```

Comment:
Use the position instead of `player_id`. The `player_id` will be constantly increasing with every game made, where as the position numbers are locked within a static range.